cdap-unit-test/src/test/java/co/cask/cdap/security/AuthorizationTest.java

```diff
@@ -543,6 +549,38 @@ public Boolean call() throws Exception {
     assertNoAccess(appId);
   }
 
+  @Test
+  public void testSearchMetadata() throws Exception {
+    createAuthNamespace();
+    deployApplication(AUTH_NAMESPACE.toId(), AllProgramsApp.class);
```

Comment:
when the app is deployed, some metadata is automatically annotated to every entity in the app. might want to test some more entities than just the stream